app/views/pictures/index.html.haml

```diff
@@ -0,0 +1,27 @@
+- if @pictures.any?
+  - @pictures.reverse.each do |picture|
+    %div.img_container
```

Comment:
`%div.img_container` can be written as `.img_container` since `%div` is implicit